django/contrib/admin/options.py

```diff
@@ -339,11 +339,11 @@ def lookup_allowed(self, lookup, value):
         # Check FKey lookups that are allowed, so that popups produced by
         # ForeignKeyRawIdWidget, on the basis of ForeignKey.limit_choices_to,
         # are allowed to work.
-        for l in model._meta.related_fkey_lookups:
+        for L in model._meta.related_fkey_lookups:
```

Comment:
A better improvement throughout the PR would be to move away from one letter variable names, e.g. `lookup` here.